CS483/CS483/Kartaclysm/Components/ComponentTrackPiece.h

```diff
@@ -31,6 +31,24 @@ namespace Kartaclysm
 		Turn
 	};
 	
+	struct OffroadSquare
+	{
+		float x1;
+		float z1;
+		float x2;
+		float z2;
```

Comment:
Declaring multiple variables on the same line is error prone and should be avoided.
Imagine you have the code `Foo* bar, baz;`.  It looks like you've declared two pointers to type Foo, but in fact you've declared a pointer to type Foo, and an instance of type Foo.